app/controllers/api/v1/planning_applications_controller.rb

```diff
@@ -37,7 +37,6 @@ def create
             constraints: constraints_array_from_param(params[:constraints]),
             planx_data: (params[:planx_debug_data].to_json if params[:planx_debug_data].present?),
             api_user: current_api_user,
-            audit_log: params.to_json,
```

Comment:
Should we double check we don't want this?

Reply:
I'll add it back and we can remove this as another PR if not needed.
Looks like it is just storing all the params from the request so not sure why it's called audit_log anyway